app/forms/instance-create.tsx

```diff
@@ -318,6 +318,7 @@ export function CreateInstanceForm() {
               memory: instance.memory * GiB,
               ncpus: instance.ncpus,
               disks: [bootDisk, ...values.disks],
```

Comment:
i'd suggest making this just `values.disks`, the fact that you can specify a disk multiple times (`boot_disk` + others, repeatedly in `disks`, some combination of both) seems to have [surprised everyone](https://github.com/oxidecomputer/omicron/pull/6585#issuecomment-2386291174). not gonna go reject this right now, but it feels like something we might reject later?

Reply:
Will do. Also need to change the mock API logic so it doesn't error when the boot disk is not in the other disks list.